compiler/src/iree/compiler/Codegen/Common/Passes.td

```diff
@@ -312,6 +312,17 @@ def HoistStaticallyBoundAllocations :
     InterfacePass<"iree-hoist-statically-bound-allocations", "mlir::FunctionOpInterface"> {
   let summary = "Hoist statically bound alloca ops to the entry block of functions";
   let constructor = "mlir::iree_compiler::createHoistStaticallyBoundAllocationsPass()";
+  let options = [
+    Option<"vscaleMin", "vscale-min", "unsigned",
+            /*default=*/"0",
+           "Minimum possible value of vscale.">,
+    Option<"vscaleMax", "vscale-max", "unsigned",
+            /*default=*/"0",
+           "Maximum possible value of vscale (a value of zero means unbounded).">
+  ];
```

Comment:
> Using 0 for unbounded (which happens to be the default) is a bit counter-intuitive for me. Would it be possible to test such case?

It's the pretty much the same as vscale_range is specified (in the LangRef). unbounded is the default behaviour because
that's saying nothing is known about vscale.